tests/notebook/test_service.py

```diff
@@ -195,3 +195,8 @@ def test_build(self):
         config = load_config_from_yaml("samples/notebook/managed-notebook/wanna.yaml", "default")
         nb_service = ManagedNotebookService(config=config, workdir=Path("."))
         assert nb_service.build() == 0
+
+    def test_sync(self):
+        config = load_config_from_yaml("samples/notebook/managed-notebook/wanna.yaml", "default")
+        nb_service = ManagedNotebookService(config=config, workdir=Path("."))
+        assert nb_service.sync(force=True) == 0
```

Comment:
I am not sure what is being tested here, besides the fact that doesn't throw errors. 

What about returning `to_be_created` and `to_be_deleted` and assert on the output. This way we can also check that the _diff functions does its job

The current assert could the `nb_service.sync(force=True) == None`(assuming no `return 0/1` is introduced)  and the effect would be the same

Reply:
Not throwing errors is already sth, as it made me change the code a bit (the .runtimes part) :)
Asserting == None is impossible in our setup, hence introducting the return 0/1
I will add the assertion on to_be_created and to_be_deleted then

Reply:
`result = nb_service.sync(force=True)`
and 

`self.assertIsNone(result)` is possible